persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/AbstractPersistentEntityRegistry.scala

```diff
@@ -123,7 +123,12 @@ abstract class AbstractPersistentEntityRegistry(system: ActorSystem, injector: I
           case other         => throw new IllegalArgumentException(s"$journalId does not support ${other.getClass.getSimpleName} offsets")
         }
         queries.eventsByTag(tag, offset)
-          .map { env => (env.event.asInstanceOf[Event], Sequence(env.offset)) }
+          .map(env =>
+            new EventStreamElement[Event](
+              PersistentEntityActor.splitPersistenceId(env.persistenceId)._2,
+              env.event.asInstanceOf[Event],
+              Sequence(env.offset)
```

Comment:
Because `env` here is `akka.persistence.query.EventEnvelope` which only supports env. Was going to write "but when we upgrade to Akka 2.4.12 ..." but then I realised that `EventEnvelope` still contains a long there, it's just the `eventsByTag` that we have changed. So I guess we need a new `EventEnvelope` as well?